src/Samples.Shared/Managers/ApiKeyManager.cs

```diff
@@ -164,7 +164,7 @@ internal static string GetDataFolder()
 #else
             string appDataFolder = Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData);
 #endif
-            string sampleDataFolder = Path.Combine(appDataFolder, "ArcGISSampleData");
+            string sampleDataFolder = Path.Combine(appDataFolder, "ESRI", "dotnetSamples", "Data");
```

Comment:
maybe slightly unrelated, but why isn't it using `DataManager.DataPath` instead of this duplicate method?

Reply:
They're slightly different but I am in favor of removing one of them. 

ApiKeyManager:

```
#if NETFX_CORE
            string appDataFolder = Windows.Storage.ApplicationData.Current.LocalFolder.Path;
#elif __ANDROID__ && !NETCOREAPP
            string appDataFolder = Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData);
#else
            string appDataFolder = Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData);
#endif
            string sampleDataFolder = Path.Combine(appDataFolder, "ArcGISSampleData");

            if (!Directory.Exists(sampleDataFolder)) { Directory.CreateDirectory(sampleDataFolder); }

            return sampleDataFolder;
```

DataManager:

```
#if NETFX_CORE
            string appDataFolder = Windows.Storage.ApplicationData.Current.LocalFolder.Path;
#elif MACCATALYST
            string appDataFolder = Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData);
#elif MAUI
            string appDataFolder = FileSystem.Current.AppDataDirectory;
#else
            string appDataFolder = Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData);
#endif
            string sampleDataFolder = Path.Combine(appDataFolder, "ArcGISSampleData");

            if (!Directory.Exists(sampleDataFolder)) { Directory.CreateDirectory(sampleDataFolder); }

            return sampleDataFolder;
```

I don't see any reason we can't just use the DataManager implementation. Unless there is something specific about bad practice storing the API key under `FileSystem.Current.AppDataDirectory` on MAUI (iOS and WinUI only).